solr/docker/build.gradle

```diff
@@ -104,6 +112,18 @@ task assemblePackaging(type: Sync) {
   into packagingDir
 }
 
+task assemblePackagingOfficial(type: Sync) {
```

Comment:
There's no need to add the docker scripts in the official packaging, those are included in the tarball now.

This can be simplified a lot, since there only needs to be one file added, the official dockerfile which is built by the `createDockerfileOfficial` task

Reply:
You are right, both official and local files pull the scripts from the tarball, thanks for this simplification.